init.lua

```diff
@@ -1,131 +1,217 @@
+	" м" .. v10 .. "д" .. v8 .. "к ",
+	" х" .. v10 .. "й ",
+	" " .. v6 .. "х" .. v10 .. "" .. v7 .. "л? ",
+	" эт" .. v6 .. " п" .. v11 .. "" .. v9 .. "д" .. v7 .. "ц ",
+	" п" .. v11 .. "" .. v9 .. "д" .. v8 .. " ",
+	" св" .. v6 .. "л" .. v6 .. "чь ",
+	" ж" .. v6 .. "п" .. v8 .. " ",
+	" г" .. v8 .. "вн" .. v6 .. " ",
+	" л" .. v6 .. "х ",
+	" г" .. v8 .. "нд" .. v6 .. "н ",
+	" " .. v10 .. "блюд" .. v6 .. "к ",
+	" ср" .. v8 .. "ть ",
+	" мн" .. v7 .. " н" .. v8 .. "ср" .. v8 .. "ть ",
+	" мн" .. v7 .. " п" .. v6 .. "х" .. v10 .. "й ",
+	" ч" .. v7 .. "рт ",
+	" тр" .. v8 .. "хн" .. v10 .. "ть ",
+	" тр" .. v8 .. "хн" .. v10 .. "л ",
+	" д" .. v7 .. "г" .. v7 .. "н" .. v7 .. "р" .. v8 .. "т ",
+	" хр" .. v7 .. "н ",
+	" х" .. v10 .. "й ",
+	" д" .. v7 .. "рьм" .. v6 .. " ",
+	" п" .. v6 .. "ш" .. v7 .. "л к ч" .. v6 .. "рт" .. v10 .. " ",
+	" мн" .. v7 .. " пл" .. v7 .. "в" .. v8 .. "ть ",
+	" х" .. v7 .. "рн" .. v12 .. " ",
+	" хр" .. v7 .. "нь ",
+	" " .. v6 .. "д" .. v11 .. "н хр" .. v7 .. "н ",
+	" н" .. v11 .. " хр" .. v7 .. "н" .. v8 .. " ",
+	" н" .. v10 .. " " .. v7 .. "г" .. v6 .. " н" .. v8 .. "хр" .. v7 .. "н ",
+	" " .. v11 .. "д" .. v11 .. " н" .. v8 .. "х" .. v7 .. "р ",
+	" н" .. v8 .. "хр" .. v7 .. "н ",
+	" п" .. v6 .. "шёл ",
+	" н" .. v8 .. "х" .. v7 .. "р ",
+	" н" .. v8 .. "хр" .. v7 .. "н ",
 
-    if uncensored == 1 then
-        minetest.kick_player(name, "Hey! Was there a bad word?")
-        minetest.log("action", "Player "..name.." warned for cursing. Chat:"..message)
-        return true
-    end
+	-- German
+	" " .. v2 .. "rschl" .. v5 .. "ch ",
+	" " .. v2 .. "rsch ",
+	" schw" .. v2 .. "nz ",
+	" w" .. v1 .. "chs" .. v4 .. "r ",
+	" schl" .. v2 .. "mp" .. v4 .. " ",
+	" h" .. v3 .. "rr" .. v4 .. " ",
+	" f" .. v1 .. "ck d" .. v1 .. "ch ",
+	" m" .. v3 .. "tt" .. v4 .. "rf" .. v1 .. "ck" .. v4 .. "r ",
+	" h" .. v3 .. "r" .. v4 .. "ns" .. v5 .. "hn ",
+	" m" .. v2 .. "st" .. v3 .. "b" .. v1 .. "r" .. v4 .. "n ",
+	" sch" .. v4 .. "" .. v1 .. "d" .. v4 .. " ",
+	" g" .. v4 .. "schl" .. v4 .. "chtsv" .. v4 .. "rk" .. v4 .. "hr ",
+	" f" .. v5 .. "tz" .. v4 .. " ",
+	" m" .. v1 .. "ssg" .. v4 .. "b" .. v3 .. "rt ",
+	" m" .. v1 .. "ssg" .. v4 .. "b" .. v3 .. "rt ",
+	" v" .. v5 .. "ll" .. v1 .. "d" .. v1 .. "" .. v5 .. "t ",
+	" brüst" .. v4 .. " ",
+	" t" .. v1 .. "tt" .. v4 .. "n ",
+	" sch" .. v4 .. "" .. v1 .. "ß" .. v4 .. " ",
+	" sch" .. v4 .. "" .. v1 .. "ss" .. v4 .. " ",
+}
 
-    if uncensored == 2 then
-        minetest.kick_player(name, "Cursing or words, inappropriate to game server. Kids may be playing here!")
-        minetest.chat_send_all("Player <"..name.."> warned for cursing" )
-        minetest.log("action", "Player "..name.." warned for cursing. Chat:"..message)
-        return true
-    end
+-- Returns true if a curse word is found
+function chat_anticurse.is_curse_found(name, message)
 
+	local checkingmessage=string.lower( name.." "..message .." " )
```

Comment:
Codesyle
```suggestion
	local checkingmessage = string.lower( name.." "..message .." " )
```

Reply:
This was how the original author typed the code. Fixed.